pytorch_lightning/utilities/enums.py

```diff
@@ -256,3 +257,22 @@ def interactive_compatible_types() -> List["_StrategyType"]:
     def is_interactive_compatible(self) -> bool:
         """Returns whether self is interactive compatible."""
         return self in _StrategyType.interactive_compatible_types()
+
+
+class _AcceleratorType(LightningEnum):
```

Comment:
Why is this protected now?

Reply:
Related discussion: https://github.com/PyTorchLightning/pytorch-lightning/pull/10505#discussion_r748313996